README.md

```diff
@@ -99,12 +101,22 @@ Get source code, run installer and follow on screen instructions:
 git clone https://github.com/AdnanHodzic/auto-cpufreq.git
 cd auto-cpufreq && sudo ./auto-cpufreq-installer
 ```
+### Update using auto-cpufreq-installer
+* The feature is available from version *1.9.8*. If your current version is below the same, Get source code, run installer and follow on screen instructions:
+
+    ```
+    git clone https://github.com/AdnanHodzic/auto-cpufreq.git
+    cd auto-cpufreq && sudo ./auto-cpufreq-installer
+    ```
+        and choose 'Update' option
+
+For further information: [--update](#update---auto-cpufreq-update)
 
 In case you encounter any problems with `auto-cpufreq-installer`, please [submit a bug report](https://github.com/AdnanHodzic/auto-cpufreq/issues/new).
 
 ### AUR package (Arch/Manjaro Linux)
 
-*AUR is currently unmaintained & has issues*! Until someone starts maintaining it, use the [auto-cpufreq-installer](https://github.com/AdnanHodzic/auto-cpufreq#auto-cpufreq-installer) if you intend to have the latest changes as otherwise you'll run into errors, i.e: [#471](https://github.com/AdnanHodzic/auto-cpufreq/issues/471). However, if you still wish to use AUR then follow the [Troubleshooting](#aur) section for solved known issues.
+*AUR is currently unmaintained & has issues*! Until someone starts maintaining it, use the [auto-cpufreq-installer](https://github.com/AdnanHodzic/auto-cpufreq#auto-cpufreq-installer) if you intend to have the latest changes as otherwise you'll run into errors, i.e: [#471](https://github.com/AdnanHodzic/auto-cpufreq/issues/471). However, if you still wish to use AUR then follow the [Troubleshooting](https://github.com/AdnanHodzic/auto-cpufreq/#aur) section for solved known issues.
```

Comment:
Let's be consistent and update `[Troubleshooting](https://github.com/AdnanHodzic/auto-cpufreq/#aur)` to new self referencing README structure i.e: (#aur)

Reply:
done and edited ``` [auto-cpufreq-installer](https://github.com/AdnanHodzic/auto-cpufreq#auto-cpufreq-installer)  to ``` [auto-cpufreq-installer](#auto-cpufreq-installer) ```

Reply:
in addition to it i wonder, why didn't I saw those changes and prompts when I pulled the latest commit.
hope it didn't messed with other parts of code

Reply:
Yes![ These changes were ](https://github.com/AdnanHodzic/auto-cpufreq/commit/faf7134cb6e3ced7205f00ec0a0ca8679d1a735f)merged with `master` in meantime which lead to these changes in README.